### PR TITLE
Unified logging search options layout

### PR DESCRIPTION
**Pull request by bzapico**
#### What does this PR do?
Base on[ the design]( https://xd.adobe.com/view/5f53b9d7-d068-4f5b-4901-f3d638308766-2962/screen/2c5f7a44-7168-4630-a623-2571ab5b615c/Logs-OPT2-collapsable-opened-alternative-order-2?x_product=cc-slack%2F1.4.0)

-Search sidebar styles
-Some interaction to show small search sidebar styles
- integration in common view

#### Where should the reviewer start?
Checking the new components hosted in logs folder

#### What is missing?
- Input for dropdown (will be design in another task)
- The logic for buttons (will be design in another task)

#### How should this be manually tested?
Running the app and navigating to logs. Is a mocked view

#### What are the relevant tickets?

- [NP-2415](https://nalej.atlassian.net/browse/NP-2415)

#### Screenshots (if appropriate)
Captura de pantalla 2019-12-16 a las 10 37 06

### CODE REVIEW

File: src/app/scss/base/_helpers.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-@import 'app/scss/variables/mixins';
+// @import 'app/scss/variables/mixins';
```
Comment (jrodva) [inline src/app/scss/base/_helpers.scss]: @bzapico We should avoid commented code

File: src/app/scss/variables/_mixins.scss [hunk 2]
```diff
@@ -70,50 +60,24 @@
...
-	@media all and (min-width: 2560px), all and (min-height: 1021px) {
-		@content;
-	}
-}
 @mixin for-big-desktop-up {
 	@media all and (min-width: 2560px) {
 		@content;
 	}
 }
+// $font-12: 12px;
```
Comment (jrodva) [inline src/app/scss/variables/_mixins.scss]: @bzapico We should avoid commented code

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -539,4 +561,4 @@
 		"tableTargetInstance": "Target Instance",
 		"tableType": "Type"
 	}
-}
+}
```
Comment (jrodva) [inline src/assets/i18n/en.json]: We should have a line break here.

File: src/index.html [hunk 4]
```diff
@@ -18,21 +18,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Nalej</title>
     <base href="/">
-  
+
```
Comment (jrodva) [inline src/index.html]: @bzapico We should avoid unnecessary blank lines.

File: src/app/logs/search-logs/search-logs.component.ts [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+import { TranslateService } from '@ngx-translate/core';
+import { FormGroup, FormControl, FormBuilder } from '@angular/forms';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+
+@Component({
+  selector: 'search-logs',
+  templateUrl: './search-logs.component.html',
+  styleUrls: ['./search-logs.component.scss'],
+  animations: [
+    trigger('slideInOut', [
```
Comment (jrodva) [inline src/app/logs/search-logs/search-logs.component.ts]: A little reminder, if we decide finally use a css animation, we have to delete this.

File: src/app/scss/variables/_mixins.scss (hunk 2 above)
Comment (bzapico) [reply src/app/scss/variables/_mixins.scss]: yes!

File: src/app/scss/base/_helpers.scss (hunk 1 above)
Comment (bzapico) [reply src/app/scss/base/_helpers.scss]: yes!

File: src/assets/i18n/en.json (hunk 3 above)
Comment (bzapico) [reply src/assets/i18n/en.json]: Done!

File: src/index.html (hunk 4 above)
Comment (bzapico) [reply src/index.html]: Done

File: src/app/logs/search-logs/search-logs.component.ts (hunk 5 above)
Comment (bzapico) [reply src/app/logs/search-logs/search-logs.component.ts]: ok!

**Pull request merged**